charts/cluster-autoscaler/README.md

```diff
@@ -326,7 +328,14 @@ For Kubernetes clusters that use Amazon EKS, the service account can be configur
 
 In order to accomplish this, you will first need to create a new IAM role with the above mentions policies.  Take care in [configuring the trust relationship](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html#iam-role-configuration) to restrict access just to the service account used by cluster autoscaler.
 
-Once you have the IAM role configured, you would then need to `--set rbac.serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=arn:aws:iam::123456789012:role/MyRoleName` when installing.
+Once you have the IAM role configured, you would then need to `--set rbac.serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=arn:aws:iam::123456789012:role/MyRoleName` when installing. Alternatively, you can embed templates in values (ignore escaping characters):
```

Comment:
Can you clarify why we are including escape characters in the example on L337 and telling the user to ignore them?

Reply:
@jackfrancis other parts of the README template (that I did not modify) uses values examples, i.e: `{{ }}` that requires to be escaped other wise helm-docs will fail trying to resolve them. The comment to ignore them is to avoid chart users literally copy-pasting escaping characters

Reply:
Example: https://github.com/kubernetes/autoscaler/blob/4495b04278b24966c7bc4aa9ffe025cc0d702eaf/charts/cluster-autoscaler/README.md#templating-the-autodiscoveryclustername

Reply:
Thx for the clarification! I think this is a better way to handle this, updating the existing instance that you copied:

- https://github.com/kubernetes/autoscaler/pull/7994

Reply:
no problem at all @jackfrancis ! PR should be updated now